src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] colors = {"Blue", "Green", "Yellow", "Red", "Black", "Brown", "Violet"};
+
+    public String[] getColors() {
+        return colors;
+    }
+
+    public void setColors(String[] colors) {
+        this.colors = colors;
+    }
```

Comment:
let's remove getters and setters from this class

Reply:
done